src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -269,8 +277,14 @@ public void testExistingTargetFileXmlOutput() throws Exception {
     public void testExistingTargetFilePlainOutput() throws Exception {
 
         exit.checkAssertionAfterwards(() -> {
-            assertEquals(String.format(Locale.ROOT, "Starting audit...%n"
-                    + "Audit done.%n"), systemOut.getLog());
+            final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_STARTED_MESSAGE,
+                    null, null, getClass(), null);
+            final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_FINISHED_MESSAGE,
+                    null, null, getClass(), null);
```

Comment:
@subkrish , please move 
```
            final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_STARTED_MESSAGE,
                    null, null, getClass(), null);
            final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_FINISHED_MESSAGE,
                    null, null, getClass(), null);
```
to be class fields, that locallized messages is not changeable, so they are like a constants.